Backends/CLX/input.lisp

```diff
@@ -141,27 +141,27 @@
       ((:button-press :button-release)
        (let ((modifier-state (clim-xcommon:x-event-state-modifiers *clx-port* state))
              (button (decode-x-button-code code)))
-         (if (and (eq event-key :button-press)
-                  (member button '(#.+pointer-wheel-up+
-                                   #.+pointer-wheel-down+
-                                   #.+pointer-wheel-left+
-                                   #.+pointer-wheel-right+)))
-             (make-instance 'climi::pointer-scroll-event
-                            :pointer 0
-                            :button button :x x :y y
-                            :graft-x root-x
-                            :graft-y root-y
-                            :sheet sheet
-                            :modifier-state modifier-state
-                            :delta-x (case button
-                                       (#.+pointer-wheel-left+ -1)
-                                       (#.+pointer-wheel-right+ 1)
-                                       (otherwise 0))
-                            :delta-y (case button
-                                       (#.+pointer-wheel-up+ -1)
-                                       (#.+pointer-wheel-down+ 1)
-                                       (otherwise 0))
-                            :timestamp time)
+         (if (member button '(#.+pointer-wheel-up+
```

Comment:
Maybe add a comment in the code that we intend to map `:button-press` for scroll "buttons" to `pointer-scroll-event` and `:button-release` to nothing. I'm suggesting this because there has been some confusion in the past.

Reply:
I've added brief comment